.pre-commit-config.yaml

```diff
@@ -26,15 +26,6 @@ repos:
     additional_dependencies:
     - mdformat-toc
     - mdformat-tables
-- repo: https://github.com/jumanjihouse/pre-commit-hook-yamlfmt
```

Comment:
Why delete this pre-commit?

Reply:
It mangles long lines in a way that does not respect the parsed values (either by YAML or Ansible). There is no mechanism to ignore lines (https://github.com/google/yamlfmt/issues/126).

This is an example that I was able to workaround previously: 3bd171b1e3395d8257a65926287dd3df4b5d778d. Now that it's doing it again, it seems wiser to add a linter rather than auto-formatter.

Reply:
I will reduce to a nit so long as we follow up. 

I will point out that there is also a [`exclude` config](https://github.com/google/yamlfmt/blob/main/docs/config-file.md#configuration) that could be used to just drop support for this file instead of removing.